features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/blocks/application/application-view/block.jag

```diff
@@ -20,6 +21,11 @@ jagg.block("application/application-view", {
             application = result.application;
 
             var result2 = jagg.module("subscription").getAllSubscriptionsOfApplicationById(username, applicationId, 0, 10);
+            var scopesResult = jagg.module("subscription").getAllScopesOfApplicationSubscription(username, applicationId);
+
+            for (i = 0; i < scopesResult.result.size(); i++) {
```

Comment:
NPE?

Reply:
No, scopesResult is a JSON array passed from the backend.it could be either empty JSON array or filled array. It never becomes a null object.